test/GovLst.t.sol

```diff
@@ -476,6 +476,10 @@ contract Constructor is GovLstTest {
     assertEq(lst.MAX_FEE_BIPS(), 2000); // 20% in bips
   }
 
+  function test_StakeForShares() public view {
```

Comment:
Needs a more expressive name for scopelint

Reply:
Done: 3d7a520307fe8a12df63ece44b2671ceaec41bb5